lakefile.toml

```diff
@@ -0,0 +1,16 @@
+name = "aesop"
+defaultTargets = ["Aesop"]
+precompileModules = false # We would like to turn this on, but it breaks the Mathlib cache.
```

Comment:
FYI, this was fixed a while ago, so `precompileModules = true` should not break the Mathlib cache anymore.